docs/File-Formats.md

```diff
@@ -1375,3 +1376,88 @@ Example with 2 generic entities and 3 samples:
 <tr><td>mut_sig_mean</td><td>name of mean</td><td>description of mean</td><td>0.370</td><td>0.010</td><td>0.005</td></tr>
 <tr><td>mut_sig_confidence</td><td>name of confidence</td><td>description of confidence</td><td>0.653</td><td>0.066</td><td>0.050</td></tr>
 </table>
+
+## Resource Data
+
+The resource data is used to capture resouce data in patients, samples and studies.  
```

Comment:
would be good to elaborate a bit more what resources are e.g. similar to the RFC:

```
Resources are URIs that can point to:

Files: pdf, txt, png, json, etc.
Web links: non-file links e.g. URLs to other systems
```